web/Web/web/review/urls.py

```diff
@@ -0,0 +1,16 @@
+from django.urls import path
+
+from web.review import views
+
+app_name = "review"
+
+urlpatterns = [
+    path('', views.ReviewHomePageView.as_view(),
+         name='main'),
+
+    # Ajax views
+    path('get_docs/', views.DocAJAXView.as_view(),
+         name='get_docs'),
+    path('get_docs_ids/', views.DocAJAXView.as_view(),
```

Comment:
Why do we need a new url path if `get_docs` and `get_docs_ids` are calling the same view `views.DocAJAXView`?

Reply:
Removed the extra one